docs/oauth2client-deprecation.rst

```diff
@@ -0,0 +1,62 @@
+:orphan:
```

Comment:
@jonparrott Ha I know what an orphan is, what I meant was "why not include it in a toctree?"